x-pack/plugins/secops/public/components/event_details/event_fields_browser.test.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import { mount } from 'enzyme';
+import { noop } from 'lodash/fp';
+import * as React from 'react';
+import { DragDropContext } from 'react-beautiful-dnd';
+import { Provider as ReduxStoreProvider } from 'react-redux';
+import { ThemeProvider } from 'styled-components';
+
+import { mockEcsData } from '../../mock/mock_ecs';
+import { createStore } from '../../store';
+import { EventFieldsBrowser } from './event_fields_browser';
+
+describe('EventFieldsBrowser', () => {
+  describe('column headers', () => {
+    ['Field', 'Value', 'Description'].forEach(header => {
+      test(`it renders the ${header} column header`, () => {
+        const store = createStore();
+        const wrapper = mount(
+          <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
+            <ReduxStoreProvider store={store}>
+              <DragDropContext onDragEnd={noop}>
+                <EventFieldsBrowser data={mockEcsData[0]} />
+              </DragDropContext>
+            </ReduxStoreProvider>
+          </ThemeProvider>
+        );
+
+        expect(wrapper.find('thead').containsMatchingElement(<span>{header}</span>)).toBeTruthy();
+      });
+    });
+  });
+
+  describe('filter input', () => {
+    test('it renders a filter input with the expected placeholder', () => {
+      const store = createStore();
+      const wrapper = mount(
+        <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
+          <ReduxStoreProvider store={store}>
+            <DragDropContext onDragEnd={noop}>
+              <EventFieldsBrowser data={mockEcsData[0]} />
+            </DragDropContext>
+          </ReduxStoreProvider>
+        </ThemeProvider>
+      );
+
+      expect(wrapper.find('input[type="search"]').props().placeholder).toEqual(
+        'Filter by Field, Value, or Description...'
```

Comment:
Looks like this guy still needs to be i18n'd ([source](https://github.com/elastic/kibana/blob/b89fe90c38fca32f342ed2e4d8104dca3629bf0c/x-pack/plugins/secops/public/components/event_details/helpers.tsx#L27)). You can use the `mountWithIntl()` helper outlined [here](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#unit-tests) for tests.